onnxruntime/core/framework/session_state_utils.cc

```diff
@@ -96,12 +98,12 @@ static inline common::Status ExtDataTensorProtoToTensor(const Env& env, const st
   AllocatorPtr ext_data_alloc = std::make_shared<ExtDataNullAllocator>();
   const DataTypeImpl* const type = DataTypeImpl::TensorTypeFromONNXEnum(tensor_proto.data_type())->GetElementType();
   std::vector<int64_t> tensor_shape_vec = utils::GetTensorShapeFromTensorProto(tensor_proto);
-  TensorShape tensor_shape {tensor_shape_vec};
+  TensorShape tensor_shape{tensor_shape_vec};
   MLDataType ml_tensor_type = DataTypeImpl::GetType<Tensor>();
 
   auto p_tensor = std::make_unique<Tensor>(type, tensor_shape, ext_data_buf, ext_data_alloc);
```

Comment:
>ext_data_alloc [](http://example.com/codeflow?start=77&length=14)

std::move()

Reply:
I would like to limit code changes in this PR to stuff that is only relevant to the GH issue.